File: Brainz/Model/Track.cs
namespace Brainz.Model
{
    using System;
    using Newtonsoft.Json;

    public class Track
    {
        [JsonProperty("position")]
        public long Position { get; set; }

        [JsonProperty("id")]
        public Guid Id { get; set; }

        [JsonProperty("length")]
        public long Length { get; set; }

        [JsonProperty("title")]
        public string Title { get; set; }

        [JsonProperty("number")]
        public string Number { get; set; }

        [JsonProperty("recording")]
        public Recording Recording { get; set; }
    }
}

File: src/model/track.go
package model

type Track struct {
	Position        int       `json:"position"`
	ID              string    `json:"id"`
	Length          int       `json:"length"`
	Title           string    `json:"title"`
	Number          string    `json:"number"`
	Recording       Recording `json:"recording"`
	Score           float64   `json:"score"`
	AlternateTitles []string  `json:"alternate-titles"`
}
